### PR TITLE
Support Laravel 9. 

### PR DESCRIPTION
**Pull request by abrardev99**
This PR adds the new Laravel `9.x` version support to the package.

_Somehow checks are failing, I'll fix it soon_

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -14,6 +14,18 @@ jobs:
   pest:
     name: Tests (Pest)
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+      matrix:
+        os: [ubuntu-latest]
+        php: [8.0]
+        laravel: [9.*, 8.*]
+        dependency-version: [prefer-stable]
```
Comment (stancl) [inline .github/workflows/ci.yml:23]: Things that only have a single value don't need to be in the matrix. If they're needed, add them to the env.

Matrix is for things that have multiple values and become combined with each other

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -14,6 +14,15 @@ jobs:
...
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+      matrix:
+        laravel: [9.*, 8.*]
+        include:
+          -   laravel: 9.*
+              testbench: 7.*
+          -   laravel: 8.*
+              testbench: ^6.9
```
Comment (stancl) [inline .github/workflows/ci.yml:25]: What's the `include` section for when `laravel` is already a property on `matrix`?

Comment (abrardev99) [reply .github/workflows/ci.yml:25]: It adds additional configuration options to a build matrix job that already exists. For example, when running `laravel 9.*` from the matrix, use the `testbench: 7.*`

Comment (stancl) [reply .github/workflows/ci.yml:25]: Right. I'm thinking that we probably don't even need these. Composer will know which `testbench` to install when installing Laravel.

Comment (stancl) [reply .github/workflows/ci.yml:25]: Also, you only added these values to the matrix, but you're not actually using them anywhere. The script doesn't know what `laravel` or `testbench` means.

See that the action installs the same version of Laravel in [8.x](https://github.com/archtechx/helpers/runs/4856860401?check_suite_focus=true#step:3:25) and [9.x](https://github.com/archtechx/helpers/runs/4856860459?check_suite_focus=true#step:3:25)

**Comment by stancl** [comment]
Ideally, the matrix options would be named, like `L8` and `L9` instead of the `8.x` that's displayed on the screenshot below.

CleanShot 2022-01-18 at 19 06 58@2x

So perhaps we *do* need to use the matrix (to provide a name for each variant), but we don't need to specify anything other than the Laravel version.

File: composer.json [hunk 3]
```diff
@@ -19,11 +19,13 @@
...
     },
     "require-dev": {
-        "orchestra/testbench": "^6.9",
+        "orchestra/testbench": "^6.9|^7.0",
         "pestphp/pest": "^1.2",
         "pestphp/pest-plugin-laravel": "^1.0"
-    }
+    },
+    "minimum-stability": "dev",
+    "prefer-stable": true
```
Comment (stancl) [inline composer.json:30]: Why is this needed?

File: .github/workflows/ci.yml (hunk 2 above)
Comment (abrardev99) [reply .github/workflows/ci.yml]: > Right. I'm thinking that we probably don't even need these. Composer will know which testbench to install when installing Laravel.

Let me check if we do not install testbench, and it works fine. I'll remove the testbench command and _includes_.

**Comment by abrardev99** [comment]
> Ideally, the matrix options would be named, like L8 and L9 instead of the 8.x that's displayed on the screenshot below.

Yeah, I renamed it accordingly.

File: composer.json (hunk 3 above)
Comment (stancl) [reply composer.json:30]: I see 👍🏻

File: .github/workflows/ci.yml (hunk 2 above)
Comment (abrardev99) [reply .github/workflows/ci.yml]: I see. We don't need to install testbench specific version because Composer does it automatically. Also, removed it from the matrix. Now matrix has just `laravel: [9, 8]`.

File: .github/workflows/ci.yml [hunk 4]
```diff
@@ -12,13 +12,17 @@ on:
...
+    strategy:
+      fail-fast: true
+      matrix:
+        laravel: [9, 8]
 
     steps:
     - uses: actions/checkout@v2
     - name: Install composer dependencies
-      run: composer install
+      run:  composer require "illuminate/support:^${{ matrix.laravel }}.0"
```
Comment (stancl) [inline .github/workflows/ci.yml:25]:
```suggestion
      run: composer require "illuminate/support:^${{ matrix.laravel }}.0"
```
